starlette_admin/fields.py

```diff
@@ -276,6 +276,68 @@ def additional_js_links(self, request: Request, action: RequestAction) -> List[s
         return []
 
 
+@dataclass
+class SimpleMDEField(BaseField):
+    """A field that provides a Markdown editor for long text content using the
+    [SimpleMDE](https://simplemde.com/) library.
+
+    This field can be used as an alternative to the [TextAreaField][starlette_admin.fields.TextAreaField]
+    to provide a more sophisticated editor for user input.
+
+    Parameters:
+        version: SimpleMDE version
+        spell_checker: Enable spell checker
+        status: Show status bar at the bottom
+        hide_icons: Hide icons from toolbar
+        autofocus: Enable autofocus
+        other_options: Other options to pass to SimpleMDE
+    """
+
+    class_: str = "field-simplemde form-control"
+    display_template: str = "displays/simplemde.html"
+    form_template: str = "forms/simplemde.html"
+    version: str = "1.11.2"
+    placeholder: str = ""
+    spell_checker: bool = False
+    status: bool = False
+    hide_icons: List[str] = dc_field(default_factory=list)
+    autofocus: bool = True
+    options: Dict[str, Any] = dc_field(default_factory=dict)
+    """For more options, see the [SimpleMDE](https://simplemde.com/)"""
+
+    def __post_init__(self) -> None:
+        if self.options.get("placeholder", None) is None:
+            self.options["placeholder"] = self.placeholder
+        if self.options.get("spellChecker", None) is None:
+            self.options["spellChecker"] = self.spell_checker
+        if self.options.get("status", None) is None:
+            self.options["status"] = self.status
+        if self.options.get("hideIcons", None) is None:
+            self.options["hideIcons"] = self.hide_icons
+        if self.options.get("autofocus", None) is None:
+            self.options["autofocus"] = self.autofocus
+        super().__post_init__()
```

Comment:
When using this approach, any updated field attributes after initialization will be ignored. My suggestion is to construct the JSON object in the `html_params` function.